src/main/java/use_cases/display_availability_timeblocks/DisplayAvailabilityTimeBlockPresenter.java

```diff
@@ -0,0 +1,14 @@
+package use_cases.display_availability_timeblocks;
+
+public class DisplayAvailabilityTimeBlockPresenter implements DisplayAvailabilityTimeBlockOutputBoundary {
+
+    @Override
+    public String availabilityTimeBlockDisplayed(DisplayAvailabilityTimeBlockModel displayedAvailabilityTimeBlock) {
+        return displayedAvailabilityTimeBlock.toString();
```

Comment:
This is _almost_ perfect! Ideally, your model only has a constructor, attributes, and getters for those attributes. The formatting you'd put in your .toString() method is something you might instead want to implement right here in your presenter! It's the _presenter's_ job to format outgoing data; the model just stores that data.